presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -2995,7 +2997,7 @@ public Optional<ConnectorNewTableLayout> getInsertLayout(ConnectorSession sessio
 
         Optional<HiveBucketHandle> hiveBucketHandle = getHiveBucketHandle(session, table);
         if (!hiveBucketHandle.isPresent()) {
-            if (!isShufflePartitionedColumnsForTableWriteEnabled(session) || table.getPartitionColumns().isEmpty()) {
```

Comment:
I assume that we always set partition columns if isShufflePartitionedColumnsForTableWriteEnabled is true? Thanks!

Reply:
We don't set partitioning schema for non partitioned tables